### ISSUE TITLE
transformation center point when scaling shouldn't be the center of the item by default

### ISSUE THREAD

**Issue by kutoman** [UnderwaterApps/overlap2d#146]
the center point should be centered when holding **shift** imho.

**Comment by azakhary** [comment]
Oh thats a tough one.. Basically the entire transformation is VERY hard to do. It looks simple - it is super hard. It is 10 times more hard one origin point is not in the center so... eh, someone should do it some day, but it is so hard I gave up on it to do other stuff.

**Comment by kutoman** [comment]
yeah, from previous posts I got to know that :).. maybe someday I will try this task
